app/page.tsx

```diff
@@ -1,5 +1,11 @@
 import Image from "next/image";
 
 export default function Home() {
-  return <main className="h-screen w-screen bg-pink"></main>;
+ creating-feature-header
+  return (
+    <main className="h-screen w-screen">
```

Comment:
Consider moving the height and width setting to the `body` tag. On that change, also consider using `min-h-screen` for the height.